src/include/function/old_engine_string_functions.h

```diff
@@ -68,6 +68,10 @@ class OldEngineStringFunctions {
   // Upper, Lower
   static type::Value Upper(const std::vector<type::Value> &args);
   static type::Value Lower(const std::vector<type::Value> &args);
+
+  // Sequence-related
+  static type::Value Nextval(const std::vector<type::Value> &args);
```

Comment:
These two functions, Nextval() and Currval() should be put into a new sequence file and update binder to bind the file

Reply:
Correct! These are **not** string functions!